doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst

```diff
@@ -384,11 +106,10 @@ On the host, start Pacemaker.
 
     # pcs cluster start
 
-Wait for the host to become the DC. The output of ``pcs status`` should look
-as it did in `Disable Stonith and Quorum`_.
+Wait for the host to become the DC.
 
-Integrate as Guest Node
-_______________________
+Integrate Guest Node into Cluster
+_________________________________
 
 If you didn't already do this earlier in the verify host to guest connection
 section, add the KVM guest's IP address to the host's ``/etc/hosts`` file so we
```

Comment:
You added the hosts entry earlier in the walk-through, so it no longer needs to be done here